en/installconfig-storage.md

```diff
@@ -1,79 +1,75 @@
+table_of_contents: True
 
 
 # Storage
 
-MAAS has the ability to configure any storage layout during node deployment.
-MAAS doesn't just do simple partitioning - it supports complex storage layouts,
-including setting up and configuring Bcache, RAID, and LVM. It also offers
-fine-grained control over the creation, deletion, formatting and mounting of
-both block devices and partitions (see [Block devices][block-devices] and
-[Partitions][partitions]).
+The final storage configuration that a deployed node uses can be influenced
+significantly. MAAS supports traditional disk partitioning as well as more
+complex options such as LVM, RAID, and Bcache. UEFI is also supported.
 
+A node's storage is dependant upon the underlying system's disks but its
+configuration (how the disks get used) is the result of a storage template. In
+MAAS this template is called a *layout* and it gets applied to a node when it
+is commissioned.
 
-## Layouts
-
-When a node is acquired by a user it gets a default storage layout. This
-layout provides the basic storage configuration to allow a node to deploy
-successfully. The default storage layout can also be adjusted allowing an
-administrator to make the decision on which layout will be the default.
-
-The users deploying nodes are not limited by the default. They can set an
-explicit storage layout when they acquire a node or after they have acquired a
-node with the set-storage-layout API. The user acquiring a node or performing
-the set-storage-layout API calls can also customize the layout generation.
-Each layout has a set of options that can be set to adjust the generated
-layout.
-
-### LVM Layout
-
-The 'LVM Layout' creates a volume group vgroot on a partition that spans the
-entire boot disk.  A logical volume lvroot is created for the full size of the
-volume group. The lvroot is formatted with ext4 and set as the / mount point:
+Once a layout is applied, a regular user can make modifications to a node at
+the filesystem level in order to arrive at the node's final storage
+configuration.
 
+When a node is no longer needed a user can choose from among several disk
+erasure types before releasing it.
 
 
-| Name      | Size     | Type  | Filesystem Type       | Mount point  |
-|:----------|----------|-------|---------------|--------------|
-| sda       | 100G     | disk  |               |              |
-| sda1      | 512M     | part  | fat32         | /boot/efi    |
-| sda2      | 99.5G    | part  | lvm-pv(vgroot)|              |
-| vgroot    | 99.5G    | lvm   |               |              |
-| lvroot    | 99.5G    | lvm   | ext4          |  /           |
+## UEFI
 
-The following options are supported for this layout. :
+A node booting with UEFI will be accommodated by every layout type by having it
+include an EFI boot partition (`/boot/efi`) in its configuration. Other than
+setting the node to boot from UEFI, no other action is asked of the user.
```

Comment:
This makes better sense the other way around to me. ie, something more like:
Including an EFI boot partition (`/boot/efi`) in a node's configuration will allow UEFI booting to be adopted by every layout type.